docs/source/developer/unit_testing.rst

```diff
@@ -1,36 +1,39 @@
+.. _unit-testing-cmakepplang-functions:
+
 **********************************
 Unit Testing CMakePPLang Functions
 **********************************
 
 This page contains guidelines, tips, and tricks which relate to unit testing
-functionality in the CMakePPLang module. The contents of this page is somewhat
+functionality in the CMakePPLang module. The content of this page is somewhat
 exclusive to CMakePPLang and is in addition to project-wide guidelines.
 
 Testing Signatures
 ==================
 
-The CMake language is weakly typed, the CMakePP language is strongly typed.
-Because CMakePP is unit tested, users can be confident that CMakePP language 
-functions will correctly enforce type safety (and if they don't that is a bug).
-Unfortunately, this means that all other functions intended to implement
-CMakePP functions need to have their type-safety checked manually. This section
-focuses on how to unit-test the signatures of functions which are strongly
-typed, but are not CMakePP language functions.
+The CMake language is weakly typed, while CMakePPLang is strongly typed.
+Because CMakePPLang is unit tested, users can be confident that CMakePPLang
+functions will correctly enforce type safety (and if they don't, that is a
```

Comment:
Instead of functions "code"?  I think that makes it more clear this applies to say classes too.